app/assets/javascripts/listings/templates/listing/_income-table.html.slim

```diff
@@ -2,9 +2,9 @@
   table role="grid"
     thead
       tr
-        th scope="col"  household size
-        th scope="col"  maximum income/month
-        th scope="col"  maximum income/year
+        th scope="col"  {{'t.household_size' | translate}}
+        th scope="col"  {{'t.maximum' | translate}} {{'t.income_month' | translate}}
```

Comment:
these two probably need to be combined into one string, again because of word order differences between languages.